gapis/api/vulkan/api/synchronization.api

```diff
@@ -214,36 +227,75 @@ cmd void vkDestroySemaphore(
   delete(Semaphores, semaphore)
 }
 
+sub void SignalSemaphore(
+    VkSemaphore                                 semaphore,
+    u64                                         value) {
+  sem := Semaphores[semaphore]
+  if sem == null {
+    vkErrorInvalidSemaphore(semaphore)
+  } else if sem.TimelineSemaphoreInfo == null {
+    vkErrorInvalidSemaphore(semaphore)
+  } else {
+    sem.TimelineSemaphoreInfo.value = value
+    recordSemaphoreSignal(semaphore, value)
```

Comment:
Should this call queueForwardProgress()?

Reply:
Great catch, yes it should.